src/Analyser/MutatingScope.php

```diff
@@ -1977,6 +1989,25 @@ private function getNullsafeShortCircuitingType(Expr $expr, Type $type): Type
 		return $type;
 	}
 
+	private function transformVoidToNull(Type $type, Node $node): Type
+	{
+		if ($node->getAttribute(self::KEEP_VOID_ATTRIBUTE_NAME) === true) {
```

Comment:
I don't get this. So if it's true then it doesn't transform the type? It should be the opposite, right?

Reply:
No, that should be correct. It is named "keepVoid" -> don't transform it to null

Reply:
Oh alright